games-arcade/wormux/wormux-master.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Very nice game"
+DESCRIPTION="Arcade game based on Worms"
```

Comment:
This would be better for SUMMARY, and we have to find a good DESCRIPTION...

What do you think about this:

DESCRIPTION="War has been declared! Battle your favorite free software mascots in the Wormux arena. With big sticks of dynamite, grenades, baseball bats, and bazookas, you can exterminate your opponent in a 2D cartoon style scenery. The goal of the game is to destroy all of your opponents' mascots."

Reply:
That's a great idea! I'm changing it right now. Thank you for the idea :)